src/MDMMotionCurve.h

```diff
@@ -152,6 +167,15 @@ typedef NS_ENUM(NSUInteger, MDMSpringMotionCurveDataIndex) {
               friction }                    \
   }
 
+#define _MDMSpringWithInitialVelocity(mass, tension, friction, initialVelocity) \
```

Comment:
Why is there an underscore?

Do we want to mimic CoreGraphics?
MDMSpringMake() ?

Reply:
The existing API already follows this pattern so we unfortunately shouldn't deviate at this point.

Reply:
It is my understanding that `_AnythingFooBar()`  means "don't touch this". The underscore is meant to deter outsiders.

Reply:
Yeah... for now I'm not particularly happy with these APIs because we can't mark macros as deprecated. Unfortunately I don't yet know of an alternative for writing static specs yet because C functions can't be used to define static values.

Reply:
Naming is hard.

My understanding is that methods beginning with '_' are reserved by Apple and functions beginning with '_' are reserved by the compiler / system library.

I know this API has been around for a minute but it may be worth migrating the names to closer match UIKit / CoreGraphics.  Happy to discuss offline / later.

Naming is hard.